tcms/static/js/testplan_actions.js

```diff
@@ -2468,8 +2468,8 @@ function constructBatchTagProcessDialog(plan_id) {
           'name__startswith': request.term,
           'info_type': 'tags',
           'format': 'ulli',
-          'testcase__plan__pk': plan_id,
-          'field': 'name'
+          'case__plan': plan_id,
+          'field': 'tag__name'
```

Comment:
This is the result of replacing the TestTag ORM query with TestCaseTag. Using the child class also changes the names of the fields and the AJAX call needs to be updated appropriately.